src/components/map/rent-controls.tsx

```diff
@@ -54,6 +56,12 @@ const RentControls: React.FC<RentControlsProps> = ({
     stations.find(station => station.stationId === booking.stationId);
   const isOpenedStationBooked =
     booking && booking.stationId === openedStation.station.stationId;
+  const remainingBookingMinutes =
+    booking
+    ? new Date(
+      Date.parse(booking.expiryDateTime) - Date.now(),
```

Comment:
Ugh, that's really ugly (prettier does weird things sometimes). Maybe we could extract the computation here into multiple statements?

Reply:
See 3a883d47ec9e25e0d51ce7b1f7925b8586b72231.